src/uuid/model/entity/mod.rs

```diff
@@ -479,6 +479,17 @@ impl Entity {
                     return Err(EntityCheckoutRevisionError::RevisionAlreadyCheckedOut);
                 }
 
+                sqlx::query!(
+                    r#"
+                        UPDATE uuid
+                            SET trashed = false
+                            WHERE id = ?
+                    "#,
+                    revision_id
+                )
+                .execute(&mut transaction)
+                .await?;
```

Comment:
For a follow up issue: I guess a function `setState()` or `setTrashed()` would be good here...